presentation/mypage/src/main/kotlin/com/dhc/mypage/utils/SajuUtil.kt

```diff
@@ -0,0 +1,9 @@
+package com.dhc.mypage.utils
+
+import java.time.format.DateTimeFormatter
+import java.util.Locale
+
+internal object SajuUtil {
```

Comment:
```suggestion
internal object Formatter{
```
P2 : SajuUtil 이나 sajuCartDateFormat 은 너무 saju 에 묶인 듯한 이름처럼 느껴지는데, 여러곳에서 재사용하기 쉬운 범용적인 이름으로 지어주는건 어떨까?

Reply:
@JeonK1 
음 좋은데 ??